harnessed_jobs/rtm_aliveness_power_on/v0/ccs_rtm_aliveness_power_on.py

```diff
@@ -0,0 +1,209 @@
+"""
+Power-on aliveness tests script, based on Homer's
+harnessed-jobs/T08/rebalive_power/ccseorebalive_power.py script.
+"""
+import sys
+import time
+from collections import namedtuple
+import logging
+import java.lang
+from org.lsst.ccs.scripting import CCS
+
+CCS.setThrowExceptions(True)
+
+logging.basicConfig(format="%(message)s",
+                    level=logging.DEBUG,
+                    #level=logging.INFO,
+                    stream=sys.stdout)
+logger = logging.getLogger()
+
+class CcsSubsystems(object):
+    def __init__(self, subsystems=None):
+        if subsystems is None:
+            subsystems = dict(ts8='ts8', rebps='ccs-rebps')
+        for key, value in subsystems.items():
+            self.__dict__[key] = CCS.attachSubsystem(value)
+
+ChannelInfo = namedtuple('ChannelInfo', ['reb_ps_channel', 'ts8_mon_chan',
+                                         'low_lim', 'high_lim', 'chkreb'])
+
+def set_monitoring_interval(ts8, period_ms, logger=logger):
+    logger.info("Setting tick and monitoring period to %i ms" % period_ms)
+    command = "change monitor-update taskPeriodMillis %i" % period_ms
+    logger.debug(command)
+    ts8.synchCommand(10, command)
+
```

Comment:
Need to under stand the reason we need to increase this frequency.
The monitor-update task checks the channel states and reads the corresponding values at the driver level. Publication occurs only if there are some state changes.
If data publication is needed, then the "monitor-publish" frequency also needs to be updated.